game/resource/English/ability/items/tooltip_moriahs_ring.txt

```diff
@@ -0,0 +1,14 @@
+//==================================================================
+// Moriah's Ring
+//==================================================================
+
+"DOTA_Tooltip_Ability_item_moriahs_ring"                          "Moriah's Ring"
+"DOTA_Tooltip_Ability_item_recipe_moriahs_ring"                   "Moriah's Ring Recipe"
+"DOTA_Tooltip_Ability_item_moriahs_ring_Description"              "<h1>Toggle: Sangromancy</h1> When active Moriah's Ring grants %sangromancy_spell_amp%%% spell amplification but inflicts %sangromancy_self_damage%%% of the damage you deal with spells and items also to yourself"
+"DOTA_Tooltip_Ability_item_moriahs_ring_Note0"                      "Damage dealt to yourself by Sangromancy can be reduced and is of the same type as the spell that caused it"
```

Comment:
Maybe split this up into two notes and word it like Blademail. "Sangromancy self-damage is the damage done by the spell before any kind of reduction." and "Sangromancy self-damage type is the same as damage dealt by the spell."

Reply:
"Sangromancy self-damage is based off the damage done by the spell before any kind of reduction."